cmd/get/execution.go

```diff
@@ -22,24 +22,27 @@ Retrieves all the executions within project and domain.(execution,executions can
  bin/flytectl get execution -p flytesnacks -d development
 
 Retrieves execution by name within project and domain.
-
 ::
 
  bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r
 
-Retrieves execution by filters
+Retrieves all the tasks with filters.
 ::
 
- Not yet implemented
+ bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r --field-selector="execution.name=oeh94k9r2r" 
 
-Retrieves all the execution within project and domain in yaml format
+Retrieves all the tasks with limit and sorting.
+::
 
+ bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r --sort-by=created_at --limit=1 --asc
```

Comment:
Lets have some place mentioning what are fields are available for sorting or filtering.

Reply:
can we also explain that descending is the default and what "asc" references (unless you think it's obvious asc -> ascending and I'm totally off-base here)